thirdparty/giflib/dgif_lib.c

```diff
@@ -773,7 +773,7 @@ DGifDecompressLine(GifFileType *GifFile, GifPixelType *Line, int LineLen)
     }
 
     if (StackPtr != 0) {
-        /* Let pop the stack off before continueing to read the GIF file: */
+        /* Let pop the stack off before continuing to read the GIF file: */
```

Comment:
this is a third party code, I don't think our repo is the appropriate place to fix the typo. let's remove this from the PR.